### PR TITLE
PC-338

### PR DESCRIPTION
**Pull request by yurige-matrix**
Finished

### CODE REVIEW

File: src/Components/Generic/EncounterSheet/index.js [hunk 1]
```diff
@@ -33,12 +39,77 @@ const EncounterSheet = ({
...
       text: `${patient.firstName} ${patient.lastName} ${
         !isTabletMode ? `${t('Encounter date')}: ` : ''
-      } ${moment(encounter.startTime).format(formatDate)}`,
+      } ${Moment(encounter.startTime).format(formatDate)}`,
       separator: false,
       url: '#',
     },
   ];
 
+  useEffect(() => {
```
Comment (matrix-amiel) [inline src/Components/Generic/EncounterSheet/index.js:48]: I have a some thoughts about the location of this code, let's talk tomorrow...

File: src/Assets/Elements/MedicalIssues/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          <StyledTypography>{t('UNknown')}</StyledTypography>
+        )}
+      </StyledContentBlock>
+    </StyledDiv>
+  );
+};
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    languageDirection: state.settings.lang_dir,
```
[the remark this answers is not in the document]
Comment (yurige-matrix) [reply src/Assets/Elements/MedicalIssues/index.js]: Fixed

File: src/Components/Generic/EncounterSheet/EncounterForms/index.js [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
   patient,
   languageDirection,
   formatDate,
+  changeMedicalProblem,
 }) => {
-  return <StyledPatientFiles></StyledPatientFiles>;
+  const [listAllergyNew, setListAllergyNew] = useState([]);
+  const [listMedicalProblemNew, setListMedicalProblemNew] = useState([]);
+
+  const onChangeAllergyButton = () => {
```
[the remark this answers is not in the document]
Comment (yurige-matrix) [reply src/Components/Generic/EncounterSheet/EncounterForms/index.js:21]: Commented

File: src/Components/Generic/EncounterSheet/index.js (hunk 1 above)
Comment (matrix-amiel) [reply src/Components/Generic/EncounterSheet/index.js:48]: We talked you move this code into internal component in the PatientBackground and in the future task you connect it to redux, do not?

Comment (yurige-matrix) [reply src/Components/Generic/EncounterSheet/index.js]: Fixed

**Pull request merged**